recipes/xtensor/all/conanfile.py

```diff
@@ -103,6 +106,8 @@ def package_info(self):
         self.cpp_info.build_modules["cmake_find_package"] = [self._module_file_rel_path]
         self.cpp_info.build_modules["cmake_find_package_multi"] = [self._module_file_rel_path]
         self.cpp_info.names["pkg_config"] = "xtensor"
+        if self.settings.get_safe("compiler.libcxx") in ("libstdc++", "libstdc++11"):
+            self.cpp_info.defines.append("_GLIBCXX_USE_CXX11_ABI=1")
```

Comment:
Just completely remove it.
Dependencies of `xtensor` should define this variable themselves.
All other recipes do this as well.
```suggestion
```